enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/Breakout/BreakoutPicker/BreakoutPicker.tsx

```diff
@@ -25,18 +28,35 @@ export const BreakoutPickerInner = ({
   });
 
   return (
-    <BreakoutPopover
-      query={query}
-      stageIndex={stageIndex}
-      breakout={breakoutItem?.breakout}
-      breakoutIndex={breakoutItem?.breakoutIndex}
-      onAddBreakout={onAddBreakout}
-      onUpdateBreakoutColumn={(_, column) =>
-        breakoutItem?.updateBreakout(column)
-      }
-      onClose={() => onClose?.()}
-      isMetric={false}
-    />
+    <Stack spacing={0} w="18.75rem">
```

Comment:
I wonder if this 300px width should be overrideable via a className prop?

Reply:
Let's leave it for now and add it as a polish thing